resources/user-agents/browsers/safari/safari-7-0.json

```diff
@@ -30,15 +30,15 @@
           "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) *Version/#MAJORVER#.#MINORVER#* Safari/*",
           "device": "Macintosh",
           "platforms": [
-            "OSX_10_10", "OSX_10_9", "OSX_10_8",
+            "OSX_10_11", "OSX_10_10", "OSX_10_9", "OSX_10_8",
             "OSX"
           ]
         },
         {
           "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) *Version/#MAJORVER#.#MINORVER#*",
           "device": "Macintosh",
           "platforms": [
-            "OSX_10_10", "OSX_10_9", "OSX_10_8",
+            "OSX_10_11", "OSX_10_10", "OSX_10_9", "OSX_10_8",
```

Comment:
There were enough useragents in our logs with this Platform/Version combo that I felt it was worth adding this platform version back in this and the `7.1` file.